src/RecordManager/Base/Record/Lido.php

```diff
@@ -237,10 +237,22 @@ public function getTitle(
         $allTitles = [];
         foreach ($this->getTitleSetNodes() as $set) {
             foreach ($set->appellationValue as $appellationValue) {
-                if ($lang == null || $appellationValue['lang'] == $lang) {
-                    $titles[] = (string)$appellationValue;
+                if (!($title = trim((string)$appellationValue))) {
+                    continue;
+                }
+                $allTitles[] = $title;
+                if (!$lang) {
```

Comment:
This makes the loop work differently than before where we took the value even when $lang was null. Sure this works right?

Reply:
Oh yeah, if there is a lang and a no lang in the same set then its not going to be the same. I think I could actually add the no lang version also to the list.